salt/_modules/metalk8s.py

```diff
@@ -33,7 +33,7 @@ def deps_missing(*args, **kwargs):
     raise CommandExecutionError("Kubernetes python client is not installed")
 
 
-@depends('KUBERNETES_PRESENT', fallback_function=deps_missing)
+@depends(KUBERNETES_PRESENT, fallback_function=deps_missing)
```

Comment:
@slaperche-scality Weigh in please?

Reply:
@NicolasT ?

IIRC, this pattern was introduce by @sayf-eddine-scality in acb5f1665480cae91209ec4a9d7f5b6264bccc08, for `add_etcd_node`, and then reuse by @gdemonet for `wait_apiserver`.

But yeah, the fix look correct to me: according to the doc, if you pass a string it should be a module name, but here we're using a boolean so we should pass the variable.